python/shipStrawTracking.py

```diff
@@ -518,14 +518,14 @@ def extrapolateToPlane(fT,z):
             else:                                    fstate = fstate1
             zs = z
             NewPosition = ROOT.TVector3(0., 0., zs)
-            rep    = ROOT.genfit.RKTrackRep(13*cmp(fstate.getPDG(),0))
+            rep    = ROOT.genfit.RKTrackRep(13*cmp(fstate.getPDG(),0) )
             state  = ROOT.genfit.StateOnPlane(rep)
             pos,mom = fstate.getPos(),fstate.getMom()
             rep.setPosMom(state,pos,mom)
             detPlane = ROOT.genfit.DetPlane(NewPosition, parallelToZ)
-            shared = ROOT.genfit.SharedPlanePtr(detPlane)
+            sharedPlanePtr = ROOT.genfit.SharedPlanePtr(detPlane)
```

Comment:
Normally I'd probably not like this being the same as the type, but I think in this case it's fine.

Reply:
Changed.